app/pods/contact/route.js

```diff
@@ -21,6 +21,7 @@ export default class ContactRoute extends BaseRoute {
       countries: this.store.findAll('country'),
       markets: this.store.findAll('company-market'),
       companies: this.store.query('company', {
+        filter: { shownOnMainWebsite: 'true' },
```

Comment:
So 'filter' here filters out showOnMainWebsite? so changes 'true' to 'false'?

Reply:
Correct. 

Ember Data's `this.store.query('company')` fetches all companies from our database with a `GET /companies` request. Until today that was always fine. However, now this includes ICSF suppliers which should not be shown on the interflux.com website.

Therefor we'll ask our API to only return those `Company` which have `showOnMainWebsite` set to `true`.